docs/sce_file_contents.md

```diff
@@ -0,0 +1,131 @@
+
+To begin, you will need to load the `SingleCellExperiment` package and read the RDS file:
+
+```r
+library(SingleCellExperiment)
+sce <- readRDS("SCPCL000000_filtered.rds")
+```
+
+## Common fields
+
+### Expression counts
+
+For both unfiltered and filtered single-cell and single-nucleotide experiments, the primary cell by gene RNA-seq expression count data is provided as the `"counts"` assay.
+Each column represents a cell or droplet, each row a gene.
+Column names are cell barcode sequences and row names are Ensembl gene IDs. 
+This data is stored as a sparse matrix that can be accessed with the following R code:
+
+```r
+count_matrix <- counts(sce)
+```
+
+### Cell metrics
+
+Cell metrics calculated from the RNA-seq expression data are stored as a `DataFrame` in the `colData` slot, with the cell barcodes as the names of the rows.
+
+```r
+cell_metrics <- colData(sce)
+```
+
+The following per-cell data columns are included for each cell, calculated using the [`scuttle::addPerCellQCMetrics`](https://rdrr.io/github/LTLA/scuttle/man/addPerCellQCMetrics.html) function. 
+
+| Column name             | Contents                                                                          |
+| ----------------------- | --------------------------------------------------------------------------------- |
+| `sum`                   | UMI count for RNA-seq data                                                        |
+| `detected`              | Number of genes detected (gene count > 0 )                                        |
+| `subsets_mito_sum`      | UMI count of mitochondrial genes                                                  |
+| `subsets_mito_detected` | Number of mitochondrial genes detected                                            |
+| `subsets_mito_percent`  | Percent of all UMI counts assigned to mitochondrial genes                         |
+| `total`                 | Total UMI count for RNA-seq data and any alternative experiments (i.e., CITE-seq) |
+
+
+### Gene information and metrics
+
+Gene information and metrics calculated from the RNA-seq expression data are stored as a `DataFrame` in the `rowData` slot, with the Ensembl ID as the names of the rows.
+
+```r
+gene_info <- rowData(sce)
+```
+
+The following columns are included for all genes. 
+Metrics were calculated using the [`scuttle::addPerFeatureQCMetrics`](https://rdrr.io/github/LTLA/scuttle/man/addPerFeatureQCMetrics.html) function.
+
+| Column name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `gene_symbol` | [HUGO](https://www.genenames.org) gene symbol, if defined        |
+| `mean`        | Mean count across all cells/droplets                             |
+| `detected`    | Number of cells in which the gene was detected (gene count > 0 ) |
+
+### Experiment metadata
+
+Metadata associated with data processing is included in the `metadata` slot as a list.
```

Comment:
Since most of these fields reference how the sample was processed should we add in a link back to the processing information here for more information on what each of these fields/ alevin-fry parameters actually mean?